### PR TITLE
Added a section that starts to detail github actions

### PR DESCRIPTION
**Pull request by newtewt**

### CODE REVIEW

File: content/en/contribute/code/style-guide.md [hunk 1]
```diff
@@ -484,4 +484,34 @@ providing a length property for a collection class.
 
 [sideeffect]: http://en.wikipedia.org/wiki/Side_effect_(computer_science)
 
+## Github Actions
+
+### Managing Secrets
+
+Secrets are encrypted for use for things like passwords or API keys. They can be added at the org, repo, or repository level. They must be added by an admin.
```
Comment (garethbowen) [inline content/en/contribute/code/style-guide.md:491]: Duplication of "repo" and "repository"...

```suggestion
Secrets are encrypted for use for things like passwords or API keys and can be added at the org or repository level. They must be added by an admin. 
```

File: content/en/contribute/code/style-guide.md [hunk 2]
```diff
@@ -484,4 +484,34 @@ providing a length property for a collection class.
 
 [sideeffect]: http://en.wikipedia.org/wiki/Side_effect_(computer_science)
 
+## Github Actions
+
+### Managing Secrets
+
+Secrets are encrypted for use for things like passwords or API keys. They can be added at the org, repo, or repository level. They must be added by an admin. 
+See the full [documentation](https://docs.github.com/en/actions/reference/encrypted-secrets) on Github's site.
```
Comment (garethbowen) [inline content/en/contribute/code/style-guide.md:492]: I'd move this link to below the code example.

File: content/en/contribute/code/style-guide.md [hunk 3]
```diff
@@ -484,4 +484,34 @@ providing a length property for a collection class.
...
+      LAST_NAME: Octocat
+```
+
+### Third Party Actions
+
+Actions allow us to leverage code written by others to do tasks at build time. The concept is similar to NPM and packages. 
+
+Follow the github actions best practices for security purposes. The main points in the security best [practices documents](https://docs.github.com/en/actions/learn-github-actions/security-hardening-for-github-actions#using-third-party-actions) are
+
+1. Pin actions to a full length commit SHA
```
Comment (garethbowen) [inline content/en/contribute/code/style-guide.md:511]: A "why" would make this easier to remember.

```suggestion
1. Pin actions to a full length commit SHA so any malicious or buggy updates aren't silently included
```

**Review by garethbowen — CHANGES_REQUESTED** [review]
Nice work! Thanks for adding this. A few minor suggestions inline.

File: content/en/contribute/code/style-guide.md (hunk 1 above)
Comment (newtewt) [reply content/en/contribute/code/style-guide.md:491]: thanks I was pulling from this statement and didn't notice it. Now I need to know what the difference is. 

> Encrypted secrets allow you to store sensitive information in your organization, repository, or repository environments.

Comment (garethbowen) [reply content/en/contribute/code/style-guide.md]: Repo environments means one repo could have multiple environments it deploys to, for example, in CD you could have a testing environment that gets pushed nightly, and then a prod environment that gets pushed on tag. We haven't used that feature yet.

**Review by garethbowen — APPROVED** [review]
Awesome, thanks!

**Pull request merged**